### PR TITLE
Changes

### PR DESCRIPTION
**Pull request by Prometheus0000**

### CODE REVIEW

File: scripts/GT++.zs [hunk 1]
```diff
@@ -150,6 +151,13 @@ recipes.addShaped(<miscutils:blockFrameGtTumbaga> * 2, [
 [<miscutils:itemRodTumbaga>, <ore:craftingToolWrench>, <miscutils:itemRodTumbaga>],
 [<miscutils:itemRodTumbaga>, <miscutils:itemRodTumbaga>, <miscutils:itemRodTumbaga>]]);
 
+// --- Wire Factory Controller (until GT++ changes it)
+recipes.addShaped(<gregtech:gt.blockmachines:798>, [
+[<ore:plateBlueSteel>, <gregtech:gt.blockcasings:5>, <ore:plateBlueSteel>],
+[<ore:circuitElite>, <gregtech:gt.blockmachines:355>, <ore:circuitElite>],
+[<ore:plateBlueSteel>, <gregtech:gt.blockcasings:5>, <ore:plateBlueSteel>]]);
```
Comment (botn365) [inline scripts/GT++.zs:158]: way to cheap for a completly balance breaking multi

**Review by botn365 — CHANGES_REQUESTED** [review]

**Comment by Prometheus0000** [comment]
It's IV, like a bunch of the other ones? Are you suggesting we make all of them MAX (and thus impossible) like you suggested?

File: scripts/GT++.zs (hunk 1 above)
Comment (GTNH-Colen) [reply scripts/GT++.zs:158]: This puts it in line with the other multis at IV, this multi has always been out of step for no real reason. The entire multi is IV except from 4 plates. Now we can look into gating all of them at a higher tier sure but for now this fixes this discrepancy.

**Comment by Dream-Master** [comment]
i merge this now. A complet gt++ recipe rework is needed. @botn365 feel free to open some issue we can planing the changes.

**Review by Dream-Master — APPROVED** [review]

**Pull request merged**